app-modules/portal/routes/web.php

```diff
@@ -41,30 +41,35 @@
 use AidingApp\Portal\Http\Middleware\EnsureKnowledgeManagementPortalIsEmbeddableAndAuthorized;
 use AidingApp\Portal\Http\Controllers\KnowledgeManagementPortal\KnowledgeManagementPortalAuthenticateController;
 
-Route::prefix('portals')
+Route::prefix('portal')
     ->name('portal.')
     ->middleware([
         'web',
         EnsureFrontendRequestsAreStateful::class,
     ])
     ->group(function () {
+        Route::get('/knowledge-management/{any?}', function ($any = null) {
+            $url = $any ? '/portal/' . $any : '/portal';
+
+            return redirect($url, 301);
+        })->where('any', '.*');
```

Comment:
I believe this is going to need to be moved out of this prefix group and target `/portals/knowledge-management/{any?}` now that we have changed the prefix above.

Without doing so, redirects from the old URL structure do not work.

Reply:
Took care of this in [bf6a574](https://github.com/canyongbs/aidingapp/pull/204/commits/bf6a574a7a04a891890c3587ed8c2705c0ef9316)